test/javascript/src/components/ChartCard/BarChartByActionType.test.js

```diff
@@ -15,6 +15,7 @@ const stateFromServerForInfluenza = fs.readFileSync(
 
 it("BarChartByActionType has the expected 2 divs", () => {
   const initialState = JSON.parse(stateFromServerForInfluenza)
+  initialState.ui = { selectedActionTypeOrdinal: null }
```

Comment:
I think this is too much coupling, it should be stubbed out at this level and tested at the selectors level instead.

Reply:
fixed, and removed parsing of initialState everything is stubbed now